### ISSUE TITLE
import gitlab_group - cannot unmarshal Group.ldap_access of type bool

### ISSUE THREAD

**Issue by strowi** [terraform-providers/terraform-provider-gitlab#255]
Hi there,

### Terraform Version
```
~> tf version
Terraform v0.12.20
+ provider.aws v2.48.0
+ provider.gitlab v2.5.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- gitlab_group

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
provider "aws" {
  region = "eu-central-1"
}

terraform {
  backend "s3" {
    region         = "eu-central-1"
    bucket         = "XYZ"
    dynamodb_table = "XYZ-locks"
    key            = "gitlab-s3.tfstate"
  }
}

# Configure the GitLab Provider
provider "gitlab" {
  base_url = "https://gitlabURL/api/v4"
}

resource "gitlab_group" "teams" {
  for_each               = toset(["sys", "dev"])
  name                   = each.key
  path                   = each.key
  parent_id              = 0
  visibility_level       = "internal"
  request_access_enabled = "true"
  description            = "${each.key}-Repositories"
}
```

### Debug Output
https://gist.github.com/strowi/becfb1aec37317d87a74dbb1b846138e

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
gitlab_group should've been imported into state file.

### Actual Behavior
```
gitlab_group.teams["dev"]: Importing from ID "dev"...
gitlab_group.teams["dev"]: Import prepared!
  Prepared gitlab_group for import
gitlab_group.teams["dev"]: Refreshing state... [id=dev]

Error: json: cannot unmarshal number into Go struct field Group.ldap_access of type bool
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `tf import -var gitlab_token=XXX 'gitlab_group.teams["dev"]' dev`

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by jakemig** [comment]
Looks like this was resolved in v0.23.0 of xanzy/go-gitlab. I think all that needs to happen is for the version in `go.mod` to be bumped to at least 0.23.0.

References:
* https://github.com/xanzy/go-gitlab/pull/760
* https://github.com/xanzy/go-gitlab/blob/v0.23.0/groups.go#L60

**Comment by jakemig** [comment]
Looks like this was fixed in version 2.5.1.

**Comment by roidelapluie** [comment]
Thanks, feel free to reopen if that pops up again.
